src/locales/en.json

```diff
@@ -1,14 +1,14 @@
 {
   "nav": {
-    "item_files": "Files",
+    "item_files": "Drive",
```

Comment:
I would keep *Files* in that context.
But if you change the value, consider changing the key as well.